browser/ui/views/brave_javascript_tab_modal_dialog_view_views.h

```diff
@@ -45,7 +46,7 @@ class BraveJavaScriptTabModalDialogViewViews
   // This returns point in dialog host's widget coordinate.
   gfx::Point GetDesiredPositionConsideringSplitView();
 
-  raw_ref<content::WebContents> web_contents_;
+  raw_ref<content::WebContents, DanglingUntriaged> web_contents_;
```

Comment:
Added this because following tests failed locally on my Linux build:

```
SplitViewBrowserTest.JavascriptTabModalDialogView_DialogShouldBeCenteredToRelatedWebView (../../brave/browser/ui/views/frame/split_view_browsertest.cc:178)
```

```
SplitViewBrowserTest.JavascriptTabModalDialogView_DialogShouldBeCenteredToRelatedWebView_InVerticalTab (../../brave/browser/ui/views/frame/split_view_browsertest.cc:210)
```